.github/workflows/ci.yaml

```
@@ -563,13 +570,34 @@ jobs:
           EOF
           # Actually run the tests
           cd tests/integration/
-          npx jest --ci --color cascade
+          npx jest --ci --color scanner/cascade-nmap-ncrack.test.js
+          # Clean up:
+          kubectl -n cascading-tests delete scans --all
+          kubectl delete namespace cascading-tests
+      - name: "cascading Scans sslyze Integration Tests"
+        run: |
+          # We'll run these in a separate namespace so that only the cascadingRules we want to test will be used
+          kubectl create namespace cascading-tests
+          # Install declarative-subsequent-scans hook
+          helm upgrade --install dssh ./hooks/declarative-subsequent-scans/ -n cascading-tests
+          # Install unsafe-https
+          helm upgrade --install unsafe-https ./demo-apps/unsafe-https/ -n cascading-tests
```

Might be worthwhile to tell helm to wait untill the unsafe-https demo target is ready to ensure that nmap can acutally find it.

```suggestion
          helm upgrade --install unsafe-https ./demo-apps/unsafe-https/ -n cascading-tests --wait
```